packages/cli/src/helpers.mts

```diff
@@ -10,6 +10,17 @@ import { has } from 'lodash-es';
 
 const DIR_NAME = getDirNameForCurrentFile(import.meta);
 
+// helpers when when working with url to generate service
+export const GITHUB_PAT = process.env.GITHUB_PAT;
+const ORGANIZATION = process.env.ORGANIZATION || 'harness';
+const REPO = process.env.REPO || 'harness-core';
+const DEFAULT_BRANCH = process.env.DEFAULT_BRANCH || 'develop';
```

Comment:
Let's change the variable name to BRANCH, and make `master` as default because in all github repositories, `master` is the initial branch by default

```suggestion
const BRANCH = process.env.BRANCH || 'master';
```